plugins/notifications-backend/src/service/handlers.ts

```diff
@@ -77,6 +77,8 @@ export function getNotifications(
 
   const query = dbClient('messages').select('*');
 
+  query.orderBy('created', 'asc');
```

Comment:
I prefer we add it to the doc and create a story to add order by fields and not add a hard coded value right now.
It will also ease merging. Cause I am working on the same code.

Reply:
We can create a story for sorting as it will be needed anyway.

However this change here is meant as a fix for pagination - there is recently no guarantee about the order.

Reply:
> We can create a story for sorting as it will be needed anyway.
> 
> However this change here is meant as a fix for pagination - there is recently no guarantee about the order.

fix what? we don't really have sorting right and no one is using it at the moment